agents/demo/demo.ipynb

```diff
@@ -0,0 +1,970 @@
+      "\u001b[34mINFO  \u001b[0mParameter 'image' successfully set to '\"gcr.io/kubeflow-rl/agents:tf1.4.1-0208-0959-a276\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mUpdating tfjobs rl.render-0208-1334-ef07\n",
+      "\u001b[34mINFO  \u001b[0mCreating non-existent tfjobs rl.render-0208-1334-ef07\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'name' successfully set to '\"render-0208-1335-59aa\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'log_dir' successfully set to '\"gs://kubeflow-rl-kf/studies/replicated-env-comparison/kuka-0208-1010-922d\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'image' successfully set to '\"gcr.io/kubeflow-rl/agents:tf1.4.1-0208-0959-a276\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mUpdating tfjobs rl.render-0208-1335-59aa\n",
+      "\u001b[34mINFO  \u001b[0mCreating non-existent tfjobs rl.render-0208-1335-59aa\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'name' successfully set to '\"render-0208-1335-814d\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'log_dir' successfully set to '\"gs://kubeflow-rl-kf/studies/replicated-env-comparison/pendulum-0208-1010-5e21\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'image' successfully set to '\"gcr.io/kubeflow-rl/agents:tf1.4.1-0208-0959-a276\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mUpdating tfjobs rl.render-0208-1335-814d\n",
+      "\u001b[34mINFO  \u001b[0mCreating non-existent tfjobs rl.render-0208-1335-814d\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'name' successfully set to '\"render-0208-1335-721e\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'log_dir' successfully set to '\"gs://kubeflow-rl-kf/studies/replicated-env-comparison/pendulum-0208-1010-9de4\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'image' successfully set to '\"gcr.io/kubeflow-rl/agents:tf1.4.1-0208-0959-a276\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mUpdating tfjobs rl.render-0208-1335-721e\n",
+      "\u001b[34mINFO  \u001b[0mCreating non-existent tfjobs rl.render-0208-1335-721e\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'name' successfully set to '\"render-0208-1335-6d81\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'log_dir' successfully set to '\"gs://kubeflow-rl-kf/studies/replicated-env-comparison/pendulum-0208-1010-7b7f\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mParameter 'image' successfully set to '\"gcr.io/kubeflow-rl/agents:tf1.4.1-0208-0959-a276\"' for component 'agents_render'\n",
+      "\u001b[34mINFO  \u001b[0mUpdating tfjobs rl.render-0208-1335-6d81\n",
```

Comment:
All this output is distracting, was your leaving it in the notebook intentional?